scripts/build_upload.py

```diff
@@ -0,0 +1,476 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2012 - 2017, Continuum Analytics, Inc. All rights reserved.
+#
+# Powered by the Bokeh Development Team.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+''' Bokeh build and upload script for TravisCI release builds
+
+'''
```

Comment:
I mean porting the script to python (bash did the work enough well, but this is far better for others to understand what the code/process does).